cypress/e2e/GUI/customer_details.cy.js

```diff
@@ -0,0 +1,50 @@
+/// <reference types="Cypress" />
+describe('Detalhes do cliente', () => {
+    beforeEach(() => {
+        const dateInThePast = new Date(Date.UTC(1983, 6, 7))
+            cy.clock(dateInThePast)
+            cy.visit('/') 
+    });
+    context('Quando valido a saudação por default e a data', () => {
+        it('Então deve conter "Hi there" e a data "Wed Jul 06 1983"', () => {
```

Comment:
E a descrição do caso de teste algo assim:

```suggestion
        it('Exibe a saudação padrão quando o nome não é digitado', () => {
```

> Recomendo aplicar as mesmas idéias aos outros testes de GUI.